### ISSUE TITLE
redesign and move language dropdown menu to upper right hand corner

### ISSUE THREAD

**Issue by Derekf5** [getlantern/www#18]

**Comment by xiam** [comment]
It's already on the upper right corner. Making it fully customisable it's possible but it's also a bit more complicated than it may seem. I'd suggest we keep this issue open if we really want to make the select look different.

**Comment by Derekf5** [comment]
@xiam, yes this was ticketed before it moved.  Since the stock drop down is to much to change before monday, lets keep this open for when we have more time to change the CSS.
